bson/src/main/org/bson/codecs/configuration/mapper/ClassModel.java

```diff
@@ -0,0 +1,57 @@
+package org.bson.codecs.configuration.mapper;
+
+import com.fasterxml.classmate.MemberResolver;
```

Comment:
I don't want this pushed with a dependency on fasterxml, so please create a Jira about getting rid of it or shading it.